doc/articles/howto-consume-webservices.md

```diff
@@ -8,44 +8,35 @@ This how-to will walk you through the process of creating a multi-platform appli
 * Building a web service base class that supports REST service operations and then build a number of services that derived from it.
 * Using the services in a view-model.
 * Build a XAML UI that utilizes the view-model.
-* Test the application in UWP and WASM.
+* Test the application in Windows and WASM.
 
 Throughout the how-to there will be notes on recommended practices and tips that highlight resources for additional learning.
 
 > [!TIP]
 > The complete source code that goes along with this tutorial is available in the [unoplatform/Uno.Samples](https://github.com/unoplatform/Uno.Samples) GitHub repository - [TheCatApiClient](https://github.com/unoplatform/Uno.Samples/tree/master/UI/TheCatApiClient)
 
-## Prerequisites
-
-* [Visual Studio 2019 16.3 or later](http://www.visualstudio.com/downloads/)
-  * **Universal Windows Platform** workload installed
-  * **Mobile Development with .NET (Xamarin)** workload installed
-  * **ASP**.**NET and web** workload installed
-  * [Uno Platform Extension](https://marketplace.visualstudio.com/items?itemName=nventivecorp.uno-platform-addin) installed
-
-> [!Tip]
-> For a step-by-step guide to installing the prerequisites, see [Getting started on Visual Studio](https://platform.uno/docs/articles/get-started-vs.html)
+> [!TIP]
+> For a step-by-step guide to installing the prerequisites, see [Getting started on Visual Studio](https://platform.uno/docs/articles/get-started-vs-2022.html)
 
 ## Task 1 - Create a simple Uno application
 
 In this task you will create a simple Single Page App with the Uno Platform. This app - **TheCatApiClient** - will be used to demonstrate an approach to consuming REST web services using the **HttpClient** class.
 
 1. Open Visual Studio and click on **Create a new project**.
 
-    ![Visual Studio new project dialog](Assets/how-to-webservice/newproject1.PNG)
-
+    ![Visual Studio new project dialog](Assets/tutorial01/newproject1.PNG)
 1. In the **Search for Templates** search box, enter **Uno**
 
-    ![Visual Studio new project dialog searching for Uno](Assets/how-to-webservice/newproject2.PNG)
+    ![Visual Studio new project dialog searching for Uno](Assets/tutorial01/newproject2.PNG)
 
 1. In the filtered list of templates, select **Uno Platform App** and then click **Next**.
 
 1. In the **Configure your new project** window, set the **Project name** to **TheCatApiClient**, choose where you would like to save your project and click the **Create** button.
 
     ![Configure your new project dialog](Assets/how-to-webservice/newproject3.PNG)
 
-    > [!IMPORTANT]
-    > The C# and XAML snippets in this tutorial requires that the solution is named **TheCatApiClient**. Using a different name will result in build errors when you copy code from this tutorial into the solution.
+> [!IMPORTANT]
```

Comment:
Before these other steps, we are missing the Wizard section on what options to choose for this tutorial

Reply:
@agneszitte Thanks! It's done